Hymns/SearchView.swift

```diff
@@ -0,0 +1,30 @@
+import SwiftUI
+
+struct SearchView: View {
```

Comment:
I would rename this to HomeView (and the container to HomeContainer) or something like that. When you click the search bar, it should take you to a new view to actually do the search, and that one makes more sense to be the SearchView. Just bringing it up now so we don't have to refactor later

Reply:
Done, good idea!